client/layout/sidebar-v2/navigator/navigator-menu-item.tsx

```diff
@@ -41,7 +41,9 @@ export const SidebarNavigatorMenuItem = ( {
 					'is-active': isSelected,
 				} ) }
 				onClick={ () => onClickMenuItem( link ) }
+				href={ link }
 				id={ id }
+				as="a"
```

Comment:
We have the `isExternalLink` props. We can also use that to determine whether we need to open a new tab.

Reply:
Can you share more? I'm not quite sure what suggestion you're making here.

Reply:
I apologize for not being clear. It was one of the UX requirements when we have an external link to open the new tab instead of the current page, and I was referring to adding `target="_blank"` when we have the `isExternalLink` flag to true.

https://github.com/Automattic/wp-calypso/pull/83686#pullrequestreview-1705430159